explainaboard/metrics/metric.py

```diff
@@ -429,7 +431,19 @@ def calc_stats_from_data(
     def get_seed(self) -> np.random.SeedSequence:
         """Gets a numpy SeedSequence.
 
-        See Metric for the usage.
+        Returned SeedSequence can be used to construct other classes by this method.
+        See also the `Metric` docstring.
+
+        Example:
+            class Routine:
+                def __init__(self, metric: Metric):
+                    self._metric = metric
+                    seed = self._metric.get_seed().spawn(1)[0]
+                    # Supposes that a class, InnerRoutine holds its own SeedSequence.
+                    self._inner_routine = InnerRoutine(seed=seed)
+
+                def do_something(self):
+                    self._inner_routine.do_something_else()
```

Comment:
I think this example is weird: since metric is created outside and modified many times according to the invocation order of the program, it does not guarantee the reproducibility of the behavior.
What I meant in the previous comment is taking the exact seed (or None) by the class that is required to initialize some inner class. In this case, the Routine takes SeedSequence rather than Metric, and initializes Metric by itself.

Reply:
I see. I understand your point. Will update the sample code.